### PR TITLE
Document that host permissions can be used instead of the "tabs" permission

### LINKED ISSUE #1732: [WebExt] host permissions or <all_urls> permission behavior
## Acceptance criteria

1. - [ ] The listed features are documented sufficiently on MDN
2. - [ ] BCD is updated
3. - [ ] Interactive example and data repos are updated if appropriate
4. - [ ] The content has been reviewed as needed

## Features to document

1. Change in host permissions or <all_urls> permission behavior. I'm not 100% sure of what needs doing here. Investigation needed.

## Related Gecko bugs

1. - [ ] [host permissions or <all_urls> permission should grant access to privileged parts of the tabs API](https://bugzilla.mozilla.org/show_bug.cgi?id=1679688).

For folks helping with Firefox-related documentation features — make sure above AC have been done, but also:

* Set bugs to dev-doc-complete
* Add entry to Firefox release notes if feature is enabled in release
* Add entry to Firefox experimental features page if feature is not yet enabled in release

### PR DESCRIPTION
**Pull request by Rob--W**
Documentation for change from https://bugzilla.mozilla.org/show_bug.cgi?id=1679688

Fixes #1732.

### CODE REVIEW

**Review by rpl — CHANGES_REQUESTED** [review]
@Rob--W thanks for chasing down the many multiple places where we should mention this ability.

Besides the few comments below, I do have one more comment:
In the [host permissions section of the permissions doc page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#host_permissions) that we are linking in most of the changes in this PR there is a list of the extra privileges that an host permission does grant (the list that follows `The extra privileges include:`) and I was expecting an addition to that list in this PR.

I'm missing it or isn't yet part of the changes in this PR? (imho we should add an entry for this new ability in that list).

File: files/en-us/mozilla/add-ons/webextensions/api/tabs/index.html [hunk 1]
```diff
@@ -20,10 +20,10 @@
 <p>You can use most of this API without any special permission. However:</p>
 
 <ul>
- <li>To access <code>Tab.url</code>, <code>Tab.title</code>, and <code>Tab.favIconUrl</code>, you need to have the <code>"tabs"</code> <a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions">permission</a>.
+  <li>To access <code>Tab.url</code>, <code>Tab.title</code>, and <code>Tab.favIconUrl</code> (or to filter by these properties via {{WebExtAPIRef("tabs.query()")}}), you need to have the <code>"tabs"</code> <a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions">permission</a>, or have <a href="/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#host_permissions">host permissions</a> that match <code>Tab.url</code>.
 
   <ul>
-   <li>In Firefox, this also means you need <code>"tabs"</code> to {{WebExtAPIRef("tabs.query()")}} by URL.</li>
+    <li>Access to these properties by host permissions is supported since Firefox 86 and Chrome. In Firefox 85 and earlier, the "tabs" permission was required instead.</li>
```
[the remark this answers is not in the document]
Comment (Rob--W) [reply files/en-us/mozilla/add-ons/webextensions/api/tabs/index.html:26]: I am conscious of the space change, and for consistency two space changes are better than one. I'm editing the line anyway, so I didn't mind introducing the extra space

[the remark this answers is not in the document]
Comment (Rob--W) [reply files/en-us/mozilla/add-ons/webextensions/api/tabs/index.html]: Support for it was added in Chrome at some point, several years ago.

I just bisected and apparently it was in Chrome 50.

[the remark this answers is not in the document]
Comment (rpl) [reply files/en-us/mozilla/add-ons/webextensions/api/tabs/index.html]: I also prefer two spaces indentation, but I'm not sure it is worth it if the rest of the file is not consistently using the same convention. anyway it is not really a big deal.

[the remark this answers is not in the document]
Comment (rpl) [reply files/en-us/mozilla/add-ons/webextensions/api/tabs/index.html]: thanks for tracking that down.

**Review by rpl — APPROVED** [review]
lgtm, thanks Rob!

**Pull request merged**